static/js/adventure.ts

```diff
@@ -0,0 +1,94 @@
+// @ts-ignore
+// import CKEditorInspector from '@ckeditor/ckeditor5-inspector';
+
+import ClassicEditor from "./ckeditor";
+import { CustomWindow } from './custom-window';
+
+declare let window: CustomWindow;
+
+
+let $editor: ClassicEditor;
+const editorContainer = document.querySelector('#adventure-editor') as HTMLElement;
+const lang = document.querySelector('html')?.getAttribute('lang') || 'en';
+// Initialize the editor with the default language
+if (editorContainer) {
+    initializeEditor(lang);
+}
+
+function initializeEditor(language: string): Promise<void> {
+    return new Promise((resolve, reject) => {
+        if ($editor) {
+            $editor.destroy();
+        }
+
+        ClassicEditor
+            .create(editorContainer, {
+                codeBlock: {
+                    languages: [
+                        { language: 'python', label: 'Hedy', class: "language-python"},
+                    ],
+                },
+                language,
+            })
+            .then(editor => {
+                window.ckEditor = editor;
+                $editor = editor;
+                // hljs.highlightAll();
+
+                resolve();
+            })
+            .catch(error => {
+                console.error(error);
+                reject(error);
+            });
+    });
+}
+
+
+// Update the editor with new data
```

Comment:
Will you use this code in the future?

Reply:
this was a trail to highlight code, but will remove it for now and can always retrieve it back:)